src/renderer/modules/injector.ts

```diff
@@ -4,6 +4,12 @@ import type { GetButtonItem } from "../../types/coremods/message";
 import type { ContextMenuTypes, GetContextItem } from "../../types/coremods/contextMenu";
 import { addButton } from "../coremods/messagePopover";
 import { addContextMenuItem } from "../coremods/contextMenu";
+import { CommandManager } from "../apis/commands";
+import { CommandOptions, RepluggedCommand } from "src/types";
+
+// Slash command manager to be used in injector.
+
+const SlashCommandManager = new CommandManager();
```

Comment:
Each injector should get its own manager which is created on demand

Reply:
Not nor maybe but in the future we could add things to the command manager which are specific to each addon, so let's just do it now

Reply:
okay okay